test/modules/scrabble.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-env mocha */
+const mockBot = require('../mockBot')
+const assert = require('assert')
+
+describe('Scrabble module', function () {
+  const tests = {
+    'owo': '6',
+    'OwO': '6',
+    'jjj': '8',
+    'jjjj': 'Not a valid word',
+    'hello world': 'Not a valid word'
```

Comment:
You might want to test all the letters to make sure they're in the right place:
```suggestion
    'hello world': 'Not a valid word',
    'jumping': '19',
    'hay': '9'.
    'dwarves': '13',
    'flock': '14',
    'quartz': '27'
    'box': '12'
```